content/docs/reference/pkg/nodejs/pulumi/packet/_index.md

```diff
@@ -992,7 +993,7 @@ const pxe1 = new packet.Device("pxe1", {
     operatingSystem: "customIpxe",
     plan: "t1.small.x86",
     projectId: local_project_id,
-    userData: ignition_config_example.rendered,
+    userData: ignition_config_example.rendered.apply(rendered => rendered),
```

Comment:
I don't think this `apply` is needed anymore. Is your `tf2pulumi` up-to-date? (`tf2pulumi version` outputs `v0.5.1` for me).

Reply:
Hmm yeah my `tf2pulumi` hasn't been updated in a while. Let me check and update.

Reply:
Done.